### PR TITLE
Add contributor hof

### LINKED ISSUE #8: Add contributors hof
Add contributors hof at the bottom of the readme.
This website can be used to generate repository hof

https://gh-hof.netlify.com/

### PR DESCRIPTION
**Pull request by sreekaransrinath**
Fixes #8 
-

Added the Contributor Hall of Fame, as well as increased line spacing between categories, looks cleaner now. GGs!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,9 +51,21 @@ const { writeConf } = require('@gh-conf/gh-conf-write');
...
+
 ## Contributing
 
 Interested in contributing to this project?
 You can log any issues or suggestion related to this library [here](https://github.com/gh-conf/gh-conf-write/issues/new)
 
 Read our contributing [guide](CONTRIBUTING.md) on getting started with contributing to the codebase
+
+
+## Contributors Hall of Fame
```
Comment (arshadkazmi42) [inline README.md]: Let's keep the title `Contributors`

File: README.md [hunk 2]
```diff
@@ -49,9 +51,21 @@ const { writeConf } = require('@gh-conf/gh-conf-write');
...
+
+<a href="https://github.com/arshadkazmi42"><img src="https://github.com/arshadkazmi42.png" width="30" /></a>
+
```
Comment (arshadkazmi42) [inline README.md]: Remove this line break. we want to show all contributors in single line

File: README.md [hunk 3]
```diff
@@ -49,9 +51,21 @@ const { writeConf } = require('@gh-conf/gh-conf-write');
...
+<a href="https://github.com/justingolden21"><img src="https://github.com/justingolden21.png" width="30" /></a>
+
```
Comment (arshadkazmi42) [inline README.md]: Remove this line break

File: README.md [hunk 4]
```diff
@@ -49,9 +51,21 @@ const { writeConf } = require('@gh-conf/gh-conf-write');
...
+<a href="https://github.com/bojanadjordjevic"><img src="https://github.com/bojanadjordjevic.png" width="30" /></a>
+
```
Comment (arshadkazmi42) [inline README.md]: Remove this line break

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Great work. Thank you for taking this up.
There is a couple of small changes needed.
Once you are done with the fixes we can get this merged

File: README.md (hunk 4 above)
Comment (sreekaransrinath) [reply README.md]: Should be done now, please check

**Review by arshadkazmi42 — APPROVED** [review]
💯

**Comment by arshadkazmi42** [comment]
Awesome. Thank you for working on this 🎉

**Pull request merged**